### ISSUE TITLE
Add: Rewind TV (any rewind station will do)

### ISSUE THREAD

**Issue by gameshowrecaps** [iptv-org/iptv#11113]
### Channel ID

WPIX-DT4

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by LaneSh4d0w** [comment]
The one we have in our playlists works, the only issue is that those were flagged as "timed out" so that's why they don't show in the m3u playlists being hosted on the .io page.

**Comment by AntiPontifex** [comment]
Link to stream?

**Comment by LaneSh4d0w** [comment]
http://v-ny.theus6tv.duckdns.org/hls/11.4/playlist.m3u8

**Comment by AntiPontifex** [comment]
Second link not working for me (geo-blocked?), but other link is fine. Thanks to all: included in the next release.

**Comment by LaneSh4d0w** [comment]
> Second link not working for me (geo-blocked?), but other link is fine. Thanks to all: included in the next release.

normal, it's a link tester page lol

**Comment by gameshowrecaps** [comment]
First link doesn't work for me on VLC. It doesn't load at all.

**Comment by AntiPontifex** [comment]
First link works fine for me: you may be geo-blocked.

**Comment by gameshowrecaps** [comment]
Yep. I'm not in New York nor do I live there so this might be the case.

**Comment by Sphinxroot** [comment]
https://v-ny.mybtv.net/live/565.m3u8

**Comment by gameshowrecaps** [comment]
That one is also geoblocked.

**Comment by LaneSh4d0w** [comment]
works for me.

**Comment by kaikyoofers** [comment]
Works for me too.

**Comment by gameshowrecaps** [comment]
It's probably my network connection blocking me from accessing that.

**Comment by gameshowrecaps** [comment]
Update: The stream only works on my mobile device. It does in fact work.

**Comment by freearhey** [comment]
https://v-ny.mybtv.net/live/565.m3u8 (offline)
<img width="286" alt="image" src="https://user-images.githubusercontent.com/7253922/232163162-b354790a-a9fe-405d-b9ba-2bce636d07c8.png">

**Comment by rick999999** [comment]
any working link now?

**Comment by BellezaEmporium** [comment]
> any working link now?

Not likely to be available soon, as MyBTV doesn't seem to have it nowadays.
